ffi/tests/ffi_export.rs

```diff
@@ -130,15 +130,21 @@ fn get_new_struct() -> FfiStruct {
 
 #[allow(trivial_casts)]
 fn get_new_struct_with_params() -> FfiStruct {
-    let mut ffi_struct = get_new_struct();
+    let ffi_struct = get_new_struct();
     let params = get_default_params();
 
+    let mut output = MaybeUninit::new(core::ptr::null_mut());
```

Comment:
Why are we using `MaybeUninit` in this instance?

Reply:
We could avoid `MaybeUninit` here, i just use it for convenience here as we use `MaybeUninit` everywhere in ffi tests.

Reply:
just to demonstrate how one can emulate out-pointer. User can, of course, use other means to define the pointer